code/modules/genetics/mutations/friend_bug.dm

```diff
@@ -50,7 +50,24 @@
 /datum/genetics/mutation/sproachder_friend/onMobImplant()
 	old_faction = container.holder.faction
 	container.holder.faction = "sproachder"
-		
+
 
 /datum/genetics/mutation/sproachder_friend/onMobRemove()
-	container.holder.faction = old_faction
\ No newline at end of file
+	container.holder.faction = old_faction
+
+/datum/genetics/mutation/termite_friend
+	name = "Termite Friend"
+	key = "MUTATION_TERMITE_FRIEND"
+	desc = "Causes the body to produce a light electric signal. Termites and worms can probably sense this."
+	gain_text = "Your body tingles a bit."
+	exclusive_type = MUT_TYPE_FACTION
+	instability = 50 // this can be turned into a stronger ability that allows one to summon a friendly wurm once every 10ish minutes.
+	var/old_faction
+
+/datum/genetics/mutation/termite_friend/onMobImplant()
+	old_faction = container.holder.faction
+	container.holder.faction = "wurm"
```

Comment:
this the correct faction?

Reply:
Yea I double checked it. For some reason all termites are the same faction as the wurms.

Reply:
Its also why I didn't allow it to combine with other factions like the spider roach one. Cause you get termites and wurms. Which are both pretty decent enemies.